pkg/util/azureclient/azureclient.go

```diff
@@ -32,16 +34,36 @@ func addAcceptLanguages(acceptLanguages []string) autorest.PrepareDecorator {
 			if err != nil {
 				return r, err
 			}
-			if acceptLanguages != nil {
-				for _, language := range acceptLanguages {
-					r.Header.Add("Accept-Language", language)
-				}
+			for _, language := range acceptLanguages {
+				r.Header.Add("Accept-Language", language)
 			}
 			return r, nil
 		})
 	}
 }
 
+type loggingSender struct {
+	autorest.Sender
+}
+
+func (ls *loggingSender) Do(req *http.Request) (*http.Response, error) {
+	b, _ := httputil.DumpRequestOut(req, true)
+	os.Stdout.Write(b)
```

Comment:
yikes this is going to get noisy, can't we log it in debug mode?

Reply:
ok, i see it's dead code and you have to uncomment the line below - not my favourite form of configuration